Offices/Offices.Tests/DoctorServiceTests.cs

```diff
@@ -0,0 +1,314 @@
+using AutoFixture;
+using AutoMapper;
+using MongoDB.Driver.Core.Misc;
+using Moq;
+using Offices.Application.MappingProfiles;
+using Offices.Application.Models;
+using Offices.Application.Services;
+using Offices.Data.Entities;
+using Offices.Data.Repositories.Abstractions;
+
+namespace Offices.Tests
+{
+    public class DoctorServiceTests
+    {
+        private readonly IMapper _mapper;
+
+        public DoctorServiceTests()
+        {
+            var configuration = new MapperConfiguration(cfg =>
+            {
+                cfg.AddProfile<ApplicationMappingProfile>();
+            });
+
+            _mapper = configuration.CreateMapper();
+        }
+
+        [Fact]
+        public async void GetDoctorById_Returns_Doctor()
```

Comment:
I prefer following convention for unit tests: methodName_initialConditions_whatShouldHappen. Example: IsApplicable_LoanStatusNotDefaulted_ReturnsFalse
From your name it's not obvious what happens

Reply:
Fixed